dcp_prototype/backend/wrangling/migrations/utils/util.py

```diff
@@ -1,5 +1,11 @@
+import json
+import typing
 from typing import Dict
 
+import boto3
+
+s3 = boto3.client("s3")
```

Comment:
This isn't necessary for the entire utility file, so could we keep this variable local please?